File: simple-github/src/main/java/com/androidhuman/example/simplegithub/api/model/RepoSearchResponse.java
package com.androidhuman.example.simplegithub.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RepoSearchResponse {

    @SerializedName("total_count")
    public final int totalCount;

    // GithubRepo 형태의 리스트를 표현합니다.
    public final List<GithubRepo> items;

    public RepoSearchResponse(int totalCount, List<GithubRepo> items) {
        this.totalCount = totalCount;
        this.items = items;
    }
}

File: Tech/Kotlin-SimpleGithub/simple-github/src/main/kotlin/com/androidhuman/example/simplegithub/api/model/RepoSearchResponse.kt
package com.androidhuman.example.simplegithub.api.model

import com.google.gson.annotations.SerializedName

class RepoSearchResponse(
        @field:SerializedName("total_count")
        val totalCount: Int,

        // GithubRepo 형태의 리스트를 표현합니다.
        val items: List<GithubRepo>
)